### PR TITLE
Update node curriculum to use modern import/export for JS modules

### PR DESCRIPTION
**Pull request by ahmagdy**
Instead of using the classical way with commonjs, I'm switching the examples in the curriculum to use import/export as it's natively supported now in node.js with `"type": "module"` in `package.json`

### CODE REVIEW

File: week2/exercises/04-post-endpoint.md [hunk 1]
```diff
@@ -45,7 +46,7 @@ But first, for Express to handle JSON requests, we need to add `app.use(express.
 ```js
 // Contents of app.js
 
-const express = require("express")
+import express from 'express'
 const app = express()
 const port = process.env.PORT || 3000
```
Comment (BrantalikP) [inline week2/exercises/04-post-endpoint.md:51]: I think we should leave the default port in case someone forgets to include .env file

**Review by BrantalikP — APPROVED** [review]

File: week2/exercises/04-post-endpoint.md (hunk 1 above)
Comment (ahmagdy) [reply week2/exercises/04-post-endpoint.md:51]: Good point. It's unchanged, GH is just showing it in a weird way 😅

Comment (BrantalikP) [reply week2/exercises/04-post-endpoint.md:51]: haha, got it. Good job 💪

**Review by shpomp — APPROVED** [review]
👏🏽

**Pull request merged**